sklearn/datasets/descr/diabetes.rst

```diff
@@ -2,7 +2,37 @@ Diabetes dataset
 
 Notes
 -----
+
 Ten baseline variables, age, sex, body mass index, average blood
 pressure, and six blood serum measurements were obtained for each of n =
 442 diabetes patients, as well as the response of interest, a
 quantitative measure of disease progression one year after baseline.
+
+Note that the 10 x variables have been standardized to have mean 0 and squared length = 1 (sum(x^2)=1).
+
+Data Set Characteristics:
+
+  :Number of Instances: 442
+
+  :Number of Attributes: First 10 columns are numeric predictive values
+
+  :Target: Column 11 is a quantitative measure of disease progression one year after baseline
+
+  :Attributes:
+    :Age:
+    :Aex:
+    :Body mass index:
+    :Average blood pressure:
+    :S1:
```

Comment:
Yeah, was thinking about this too. They're both obscure but I felt as though `S1 S2 S3 S4 S5 S6`, were at least more memorable and consistent with the literature posted. I *think* `tc` is total cholesterol, `ldl` is low-density lipoprotein (thanks Google), etc., but I didn't want to make any assumptions here because I'm no medical expert either.